src/ploneintranet/library/browser/templates/library.html

```diff
@@ -158,6 +159,11 @@ <h3 tal:content="child/title">page title</h3>
 
         </div>
 
+        <p tal:condition="not:view/enabled">
```

Comment:
@gyst When calling the library, enable is not present and so the view gives a 500. Either a tal:on-error="nothing" or have the enabled everywhere. What do you think?